test/plugin/test_in_snmptrapalert.rb

```diff
@@ -2,77 +2,107 @@
 
 class SnmptrapalertInputTest < Test::Unit::TestCase
 
-    setup do
-        Fluent::Test.setup
-    end
+  setup do
```

Comment:
Don't change whitespaces. It makes it very hard to review

Reply:
I'm working on fixing it